README.rst

```diff
@@ -230,14 +230,92 @@ Choose from the list of available rules:
 
 * **binary_operator_spaces** [@Symfony]
 
-  Binary operators should be surrounded by at least one space.
+  Binary operators should be surrounded by space as configured.
 
   Configuration options:
 
-  - ``align_double_arrow`` (``false``, ``null``, ``true``): whether to apply, remove or
-    ignore double arrows alignment; defaults to ``false``
-  - ``align_equals`` (``false``, ``null``, ``true``): whether to apply, remove or ignore
-    equals alignment; defaults to ``false``
+  - ``!=`` (``'align'``, ``'align_single_space'``, ``'align_single_space_minimal'``,
```

Comment:
What about grouping all of these inside a single option that accepts an array with operators as keys and spacing to apply as value?